src_Core/RISCY_OOO/procs/RV64G_OOO/FetchStage.bsv

```diff
@@ -184,7 +184,7 @@ typedef struct {
     Bool  cap_mode;
 } PcCompressed deriving(Bits,Eq,FShow);
 function PcCompressed compressPc(PcIdx i, CapMem a) =
-    PcCompressed{idx: i, lsb: truncate(a), cap_mode: !getIntMode(a).value}; // Exact fail should be impossible
+    PcCompressed{idx: i, lsb: truncate(a), cap_mode: !getUnlegalisedIntMode(a)}; // Exact fail should be impossible
```

Comment:
The comment no longer makes sense. You could delete it or replace it with `mode must be legal if we've fetched`

Reply:
(And all the subsequent copies!)